scripts/find-apps.sh

```diff
@@ -72,6 +72,9 @@ describe_app() {
         runner="docker"
     fi
     case "${app}" in
+        apps/emqx_bridge_kafka)
+            profile='emqx-enterprise'
+            ;;
```

Comment:
Q: I wonder why doesn't this app under `lib-ee/` then? Considering also the changes in `run.sh` there are a lot of things to keep track of in different scripts, which will eventually become quite hairy.

Reply:
Ah, thanks, missed that. 👍🏼